pkg/spec/cluster_tls.go

```diff
@@ -22,8 +22,9 @@ type TLSPolicy struct {
 }
 
 type StaticTLS struct {
-	// ServerSecretName contains peer-interface and client-interface server x509 key/cert, along with peer and client CA cert.
+	// ServerSecretName contains peer-interface and client-interface server x509 key/cert and CA cert.
```

Comment:
A better naming would be:
```
  static:
    member:
      peerSecret: etcd-peer-tls
      serverSecret: etcd-server-tls
    operator:
      clientSecret: etcd-client-tls
```

It's better to call it "serverSecret" to differentiate from "clientSecret".